core/main.py

```diff
@@ -160,6 +161,17 @@ def parse_args(args):
             statements), from the target/run directory.""",
     )
 
+    retry_failed_sub = subs.add_parser(
+        "retry-failed",
+        parents=[base_subparser],
+        help="""
+            Rerun the models that failed or were skipped on the previous run.""",
+    )
+
+    retry_failed_sub.set_defaults(
+        cls=retry_failed_task.RetryFailedTask, which="retry-failed"
+    )
+
```

Comment:
Are there any additional arguments we need for this command?

Reply:
I can't think of any 🤷‍♀